documentation/directory-structure.md

```diff
@@ -7,190 +7,260 @@ https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
-
-- [Specification index]
-- [Resource Provider list]
+This article provides a comprehensive guide to the recommended directory structure and folder layout for Azure REST API specifications within the `specification` folder. The structure outlined here is designed to support both ARM (Azure Resource Manager) and data-plane services while maintaining compatibility with both TypeSpec-based and OpenAPI-based API development workflows.
 
 > [!IMPORTANT]
-> The structure described in this article is recommended. There exist some exceptions for historical reasons.
-> These exceptions are strongly discouraged going forward.
+> The structure described in this article is the **recommended standard** for all new Azure services. While some existing services may follow legacy patterns for historical reasons, all new services must adopt this structure. We are actively working on migrating legacy patterns to align with these guidelines.
 
-## Key concepts
+## Key concepts and terminology
 
-The directory structure is a reflection of a few key concepts.
+To better understand the directory structure, it's essential to familiarize yourself with these key concepts as defined in the [glossary]:
 
-Familiarize yourself with the concepts of `service`, `service group`, `grouping directory` and `uniform versioning`
-as explained in the [glossary].
+- **Service**: A customer-facing Azure service (e.g., Azure Key Vault, Azure Container Registry)
+- **Organization**: The Azure team or group responsible for one or more services
+- **Resource Provider Namespace (RPNS)**: The ARM namespace for resource management operations (e.g., `Microsoft.KeyVault`)
+- **API Version**: All Azure services use the YYYY-MM-DD(-preview) format for consistent versioning
 
-## `specification` folder
+**Additional resources:**
+- [Uniform Versioning] - Azure service versioning guidelines for consistent service evolution
+- [Specification index] - Complete list of all specifications
+- [Resource Provider list] - Official list of ARM Resource Providers
 
-The `specification` folder is the root folder for all `service` specifications.
+## Overview of the folder structure
 
-Each child of the `specification` folder corresponds to `<azureTeam>`.
-All the `service` (see [glossary]) specifications owned by that team are rooted in their `<azureTeam>` folder.
+The `specification` folder follows a hierarchical organization designed to accommodate both ARM and data-plane services. The structure ensures clear separation between different service types while maintaining consistency across all Azure service specifications.
 
-Given `<azureTeam>` folder has following structure:
+```
+specification/
+└── <organization>/
+    ├── cspell.yaml                     # Spell-check configuration
+    ├── resource-manager/               # ARM services
+    │   └── <RPNS>/                    # Resource Provider Namespace
+    │       └── <service>/             # Individual service
```

Comment:
how about using the below to indicate that we allow multiple services under namespace folder,  similar for data plane as we allow multiple services existing in the folder.
  `<service1>\ ` 
  `<service2> \ `